Parceljs cannot compile

Server running at http://localhost:1234
🚨  /Volumes/HDD/project/projects/lawson_event_frontend/node_modules/react-router-transition/lib/react-router-transition.js:117:27: Cannot resolve dependency '
   module.exports = require('./factoryWithThrowingShims')();
      |                            ^
}

I'm having a similar error:

`Error: Cannot find module './factoryWithThrowingShims' from '/path/to/project/node_modules/react-router-transition/lib'`

I have `prop-types` installed.

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -45,10 +45,4 @@ module.exports = {
       'node_modules',
     ],
   },
-
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'development'),
-    }),
-  ],
 };

